helm/hpcc/templates/_helpers.tpl

```diff
@@ -857,8 +860,8 @@ Generate list of available services
 {{- range $.Values.dafilesrv -}}
  {{- if not .disabled -}}
 - name: {{ .name }}
-  type: dafilesrv
-  port: {{ .servicePort | default 7600 }}
+  type: {{ .application | default "stream" }}
+  port: {{ .service.servicePort | default 7600 }}
```

Comment:
Can service be null?  Does it need
$service := .service | default dict
or similar.

Reply:
service is required, can't be missing.